website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -25,9 +25,25 @@ Multiple selections can be made within the list.
 
 ![Multiple selection example with list open](/assets/components/form/super-select/super-select-multiple-variant.png =504x*)
 
+## Trigger default states
+
+### Empty
+The default option, where the trigger is empty, showing nothing selected.
```

Comment:
@KristinLBradley Is this accurate that this option is the default one for our trigger's behavior?

Reply:
Yes, although we could change this to set our own default if desired.

<img width="348" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/effc3338-8515-43eb-bbe9-9d8cbdcc7171">

Reply:
I think this as a default works.